share/p2p/shrexnd/params.go

```diff
@@ -0,0 +1,18 @@
+package shrexnd
+
+import (
+	logging "github.com/ipfs/go-log/v2"
+
+	"github.com/celestiaorg/celestia-node/share/p2p"
+)
+
+const protocolString = "/shrex/nd/0.0.1"
+
+var log = logging.Logger("shrex/nd")
+
+// Parameters is the set of parameters that must be configured for the shrex/eds protocol.
+type Parameters = p2p.Parameters
```

Comment:
why do we have to alias?

Reply:
Relating to higher level p2p pkg inside shrex and in nodebuilder feels confusing, so I done it to make imports and usages to look nicer.

Reply:
okay